newIDE/app/src/ProjectsStorage/ResourceFetcher/index.js

```diff
@@ -39,14 +57,47 @@ type UseResourceFetcherOutput = {|
    * and must optionally be fetched by the storage provider (e.g: a URL to be downloaded).
    */
   ensureResourcesAreFetched: (
-    options: FetchAllProjectResourcesOptionsWithoutProgress
+    options: CleanAndFetchAllProjectResourcesOptionsWithoutProgress
   ) => Promise<void>,
   /**
    * Render, if needed, the dialog that will show the progress of resources fetching.
    */
   renderResourceFetcherDialog: () => React.Node,
 |};
 
+export const cleanUpResourceNames = ({
```

Comment:
I've tried creating this function to call it in both fetchers.
The goal is to clean up the names of the resources (and remove the possible unwanted token & sha) before asking the fetcher to fetch it.
Wdyt of the strategy? @4ian